src/login/components/SwamidErrors/Errors.js

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from "react";
+import { useLocation } from "react-router-dom";
+import "../../styles/index.scss";
+import i18n from "../../translation/InjectIntl_HOC_factory";
+import { swamidErrorData } from "./swamidErrorData";
+
+function Errors(props){
+  let query = new URLSearchParams(useLocation().search);
+  const [techInformation, setTechInformation] = useState({});
+  useEffect(()=> {
+    let errorurl_ts = query.get("errorurl_ts");
+    let errorurl_rp = query.get("errorurl_rp");
+    let errorurl_tid = query.get("errorurl_tid");
+    let errorurl_ctx = query.get("errorurl_ctx");
+    //Convert unix time stamp to (YYYY-MM-DD) format
+    const convertUnixDate = new Date(errorurl_ts * 1000);
+    const newDateUrlTs = 
+      ' ('+ convertUnixDate.getFullYear()+'-' + ((convertUnixDate.getMonth()+1).toString())+ '-'+(convertUnixDate.getDate().toString())+')';
+
+      setTechInformation({
+        errorurl_ts: errorurl_ts &&  errorurl_ts !== "ERRORURL_TS" ? errorurl_ts + newDateUrlTs : undefined, 
+        errorurl_rp: errorurl_rp && errorurl_rp !== "ERRORURL_RP" ? errorurl_rp : undefined,
+        errorurl_tid: errorurl_tid && errorurl_tid !== "ERRORURL_TID" ? errorurl_tid : undefined,
+        errorurl_ctx: errorurl_ctx && errorurl_ctx !== "ERRORURL_CTX" ? errorurl_ctx : undefined,
+    })
+  }, []);
+
+  let isSpecificError = "";
+  let errorurlCode = query.get("errorurl_code");
+  let errorurlCtx = query.get("errorurl_ctx");
+  let errorurlRp = query.get("errorurl_rp");
+  let common = swamidErrorData.common;
+  let specialRp = swamidErrorData["sp.ladok.se"];
```

Comment:
We can't do this for all the special SPs, there are an unknown number of them.

Reply:
I added logic for checking special SPs
```
  Object.keys(swamidErrorData).map((key) => {
    if(errorurlRp && errorurlRp.includes(key))
      return specialRPMessages = swamidErrorData[errorurlRp]
  });
```